data/djangocon-2015/videos/building-theatlantic-com-homepages-wysiwyg-admin.json

```diff
@@ -1,7 +1,7 @@
 {
   "category": "",
   "slug": "building-theatlantic-com-homepages-wysiwyg-admin",
-  "title": "Building theatlantic.com homepage’s WYSIWYG admin with Django and Knockout",
+  "title": "Building theatlantic.com homepage\u2019s WYSIWYG admin with Django and Knockout",
```

Comment:
The original character is U+2019 (was `\u2019` in the JSON)
`'` is a simplification, but it's also used for other titles.
Not a big deal in either cases :)